### PR TITLE
refactor: update collector service interfaces

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/modules-old/EditScheduleModal.vue [hunk 1]
```diff
@@ -318,7 +316,7 @@ export default {
         };
         const addSchedule = async () => {
             try {
-                const params: ScheduleAddParameter = {
+                const params: any = {
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/modules-old/EditScheduleModal.vue:319]: Question: Why isn't the type specified here? Can it accept values of multiple types?🧐

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 2]
```diff
@@ -195,9 +201,9 @@ const storeState = reactive({
 
 const state = reactive({
     current: computed(() => i18nDayjs.value.tz(i18nDayjs.value(), storeState.timezone)),
-    schedule: computed(() => collectorPageState.schedules.find((schedule) => schedule.collector_info.collector_id === props.item.collectorId)),
-    collectorState: computed(() => state.schedule?.collector_info.state),
-    toggleStatus: computed(() => (state.collectorState === COLLECTOR_STATE.ENABLED ? 'ON' : 'OFF')),
+    schedule: computed(() => collectorPageState.schedules.find((schedule) => schedule.collector_id === props.item.collectorId)),
+    scheduleState: computed<CollectorScheduleState>(() => state.schedule?.collector_info.state),
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:205]: 🥰: Thank you for applying it!

**Review by skdud4659 — APPROVED** [review]
LGTM!!!! I left a question!

**Review by sulmoJ — APPROVED** [review]
All the old types have been erased, so it's clean!
LGTM!

File: apps/web/src/services/asset-inventory/collector/modules-old/EditScheduleModal.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/modules-old/EditScheduleModal.vue:319]: Oh this file is gonna be deprecated!

**Pull request merged**